From fce0f5155bdbe86be042637045243f2325b09944 Mon Sep 17 00:00:00 2001
From: Chandrakala Chavva <Chandrakala.Chavva@cavium.com>
Date: Sun, 31 May 2020 17:48:40 -0700
Subject: [PATCH 1442/1921] octeontx2-serdes: Fix parameter passed to
 start_prbs().

Change-Id: Ib70f8b50412d79f1e868270be2f0a48cfa0251d8
Signed-off-by: Chandrakala Chavva <Chandrakala.Chavva@cavium.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/29238
Reviewed-by: Chandrakala Chavva <cchavva@marvell.com>
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Signed-off-by: Witold Sadowski <wsadowski@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/50021
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/soc/marvell/octeontx2-serdes/serdes_debugfs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/marvell/octeontx2-serdes/serdes_debugfs.c b/drivers/soc/marvell/octeontx2-serdes/serdes_debugfs.c
index 793b9fa03458..892692647411 100644
--- a/drivers/soc/marvell/octeontx2-serdes/serdes_debugfs.c
+++ b/drivers/soc/marvell/octeontx2-serdes/serdes_debugfs.c
@@ -494,7 +494,7 @@ static ssize_t serdes_dbg_prbs_write_op(struct file *filp,
 	switch (cmd) {
 	case CGX_PRBS_START_CMD:
 		arm_smccc_smc(OCTEONTX_SERDES_DBG_PRBS, cmd,
-			      qlm, mode, 0, qlm_lane, 0, 0, &res);
+			      qlm, mode, qlm_lane, 0, 0, 0, &res);
 
 		list_for_each_entry(status,
 				    &prbs_cmd_data.status_list.list,
-- 
2.31.1

